src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+public class Main {
+    public static void main(String[] args) {
+        FigureSupplier figureSupplier = new FigureSupplier();
+
+        Drawable[] figures = new Drawable[5];
```

Comment:
```suggestion
        Figure[] figures = new Figure[5];
```

5 is a magic number, create constant